addons/web/static/src/webclient/settings_form_view/widgets/mobile_apps_funnel.js

```diff
@@ -0,0 +1,33 @@
+/** @odoo-module */
+
+import { registry } from "@web/core/registry";
+
+import { Component } from "@odoo/owl";
+import { isMobileOS } from "@web/core/browser/feature_detection";
+import { Setting } from "@web/views/form/setting/setting";
+
+/**
+ * Widget in the settings that displays links to the mobile apps.
+ * As a QRCode in Desktop mode, as an image in mobile mode.
+ * Both are clickable links.
+ */
+class MobileAppsFunnel extends Component {
+    static template = "mobile_apps_funnel";
```

Comment:
```suggestion
    static template = "web.MobileAppsFunnel";
```
I know the other widgets don't follow this guideline, but they should